data/transactions/logic/evalAppTxn_test.go

```diff
@@ -440,8 +454,6 @@ func TestExtraFields(t *testing.T) {
 
 	ep, tx, ledger := MakeSampleEnv()
 	ledger.NewApp(tx.Receiver, 888, basics.AppParams{})
-	TestApp(t, "txn Sender; balance; int 0; ==;", ep)
-	TestApp(t, "txn Sender; txn Accounts 1; int 100"+pay, ep, "unauthorized")
```

Comment:
Question: this line seems to setting `txn Sender` to `itxn_field Sender`, why we are removing? Seems we will have error like `non-zero fields for type axfer`?

Reply:
I believe these lines were inadvertently included, probably by cut and paste from `TestAppPay` where they make more sense.   They are not testing the thing this test is about.

(But they are still in TestAppPay, so no test has been removed.)

Reply:
got it, I am rather lost in the test tho, sry for the questions...